src/main/java/core/basesyntax/services/impl/Parse.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.services.impl;
+
+import core.basesyntax.services.FruitParse;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Parse implements FruitParse {
```

Comment:
Class names are typically nouns or noun phrases!

Parser, not Parse
Manager, not Manage
Reader, not Read
Writer, not Write!!!
https://mate-academy.github.io/style-guides/java/java.html#s5.2.2-class-names

Reply:
Renamed class